### ISSUE TITLE
bug: Three-finger scroll is not working with VoiceOver in IOS

### ISSUE THREAD

**Issue by ethier** [driftyco/ionic#4100]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>ios</span> <span ionic-platform-version>8</span>  <span ionic-webview>webview</span>

<span ionic-description>In iOS 8, VoiceOver enabled, users are unable to three-finger swipe to scroll up/down a page, or open the menu.

Steps to reproduce:
In iOS 8 enable VoiceOver in the Accessibility menu
Open the test app
Three-finger swipe up to attempt to scroll down the page
Three-finger swipe left to attempt to open the main menu

Expected behaviour:
Three-finger swiping with VoiceOver enabled must act the same as single-finger swiping with VO disabled.

Notes:
Tested in iOS 8.4, iPhone 5S and iPhone 5C

Given that this is an issue in the native app, I've created a repo to clone that highlights the issue identified (no codepen).

With the latest ionic installed, clone the [repo](https://github.com/ethier/ionic-accessibility), add the IOS platform and deploy to a device (VoiceOver doesn't work in an emulator) to re-create this issue.</span>

<span is-issue-template></span>

**Comment by mike1e** [comment]
Our team is having this problem, also.

**Comment by Tobino** [comment]
IOS 8 still use UIWebView and not WkWebView, so native scroll event is not supported with IOS.
It seem scroll hijack catch the three-finger scroll with VoiceOver. I was able to repare the scroll by adding ```overflow-scroll="true"``` on ```<ion-content>```.

But I think Pull to Refresh, Infinite Scrolling, List Reordering, and the $ionicScrollDelegate will not work with ```overflow-scroll="true"``` on IOS.

http://blog.ionic.io/native-scrolling-in-ionic-a-tale-in-rhyme/

**Comment by rfsbsb** [comment]
hey @mike1e could you please give an example of this solution you mentioned? We are facing the same problem here but I couldn't find a way to work it around. Any help is appreciated.

**Comment by Tobino** [comment]
If you disable CSS on your ionic app, three-finger scroll works.
So I think, it's more a CSS issue on Ionic.

I use [phonegap-mobile-accessibility](https://github.com/phonegap/phonegap-mobile-accessibility) to detect screen-reader status on my app. If screen-reader is running, I add `.sr-on` on body.

I have made a little modification on my CSS:
```css
.platform-ios.sr-on .pane,
.platform-ios.sr-on .scroll-content{
	position: static;
}

.platform-ios.sr-on .scroll-content.has-header > .scroll {
	margin-top: 44px;
}
```

Three-finger scroll works, but I don't know if threre is any regression in other Ionic's feature.

**Comment by rfsbsb** [comment]
Hi @Tobino, thank you for your help. I'll give it a try today and let you know of any progress.
Thank you once again.

**Comment by rfsbsb** [comment]
Hey @Tobino just to let you know: I've tested it and worked like a charm. Indeed this kind of workaround shouldn't be needed, but glad you pointed it out. Thank you.

**Comment by ankushgoyal27** [comment]
Hi Tobino,

I tried the solution you provided. Three finger swipe is working fine, but it breaks the UI.

I am using home page and Side Navigation Menu. 

Her is the sample codepen [https://codepen.io/ankushgoyal11/pen/reQdWg](url)

Please suggest.

Thanks

**Comment by wilsolutions** [comment]
+1 having this issue to scroll with 3 fingers. It works fine on Android =/

In my case I am using the scroll in a modal view: 
<ion-modal-view id="ProductsController" class="product-modal">
  <ion-content has-header="false" scroll="false">
...
    <ion-content class="modal-content" scroll="true">
...
    </ion-content>
  </ion-content>
</ion-modal-view>

**Comment by wellsayd** [comment]
We are also having the issue with 3 finger scroll ... any suggestions?

**Comment by 200even** [comment]
We are having this issue with iOS10 WkWebView. Any better solutions than @Tobino?

**Comment by jgw96** [comment]
This issue was moved to driftyco/ionic-v1#17

**Comment by magleahy** [comment]
What happened with this? I am experiencing the same issue where cannot scroll when voiceover is on. @Tobino @ethier @mike1e @rfsbsb @ankushgoyal27?
